particle/particle/convert.py

```diff
@@ -314,10 +314,11 @@ def main(year):
     produce_files(particle2008, particlenew, year)
 
 
-def convert(fwf, latex, output):
-    table = get_from_pdg_extended(fwf,
-                                  [data.open_text(data, 'pdgid_to_latex.csv'),
-                                   latex])
+def convert(output, fwf, latex):
```

Comment:
Since you support None now, this could be optional (as you said).
```suggestion
def convert(output, fwf, latex=None):
```

Reply:
And I meant to add that. I clearly forgot in the end. Thanks for spotting it.